components/sections/Navbar/index.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Navbar from './index';
+
+type Story = StoryObj<typeof Navbar>;
+type Meta = MetaObj<typeof Navbar>;
+
+export const Default: Story = {
+  args: {
+    navItems: [
+      {
+        text: 'about',
```

Comment:
Could you write these entries as on Figma?

Learn, About, Docs, Download, Blog, Certification (in this order and this casing) (You also forgot to add the Download entry)